2021-05-01 17:57:56,957 : INFO : Start of Prediction
2021-05-01 17:57:56,957 : INFO : run_id:2021-05-01_175756_857493383
2021-05-01 17:57:56,958 : INFO : Start of Data Load, validation and transformation
2021-05-01 17:57:56,958 : INFO : Start of Archiving Old Rejected Files...
2021-05-01 17:57:56,958 : INFO : End of Archiving Old Rejected Files...
2021-05-01 17:57:56,959 : INFO : Start of Archiving Old Validated Files...
2021-05-01 17:57:56,961 : INFO : End of Archiving Old Validated Files...
2021-05-01 17:57:56,961 : INFO : Start of Archiving Old Processed Files...
2021-05-01 17:57:56,962 : INFO : End of Archiving Old Processed Files...
2021-05-01 17:57:56,962 : INFO : Start of Archiving Old Result Files...
2021-05-01 17:57:56,963 : INFO : End of Archiving Old Result Files...
2021-05-01 17:57:56,964 : INFO : Start of Reading values From Schema...
2021-05-01 17:57:56,964 : INFO : End of Reading values From Schema...
2021-05-01 17:57:56,964 : INFO : Start of Validating Column Length...
2021-05-01 17:57:56,965 : INFO : End of Validating Column Length...
2021-05-01 17:57:56,965 : INFO : Start of Validating Missing Values...
2021-05-01 17:57:56,965 : INFO : End of Validating Missing Values...
2021-05-01 17:57:56,966 : INFO : Start of Replacing Missing Values with NULL...
2021-05-01 17:57:56,966 : INFO : End of Replacing Missing Values with NULL...
2021-05-01 17:57:56,966 : INFO : Start of Creating Table...
2021-05-01 17:57:56,967 : INFO : Opened prediction database successfully
2021-05-01 17:57:57,195 : INFO : CREATE TABLE prediction_raw_data_t column_name
2021-05-01 17:57:57,268 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,369 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,478 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,544 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,620 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,688 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,755 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,821 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 17:57:57,822 : INFO : End of Creating Table...
2021-05-01 17:57:57,846 : INFO : Opened prediction database successfully
2021-05-01 17:57:57,847 : INFO : Start of Inserting Data into Table...
2021-05-01 17:57:57,847 : INFO : End of Inserting Data into Table...
2021-05-01 17:57:57,847 : INFO : Start of Exporting Data into CSV...
2021-05-01 17:57:57,847 : INFO : Opened prediction database successfully
2021-05-01 17:57:57,848 : INFO : End of Exporting Data into CSV...
2021-05-01 17:57:57,849 : INFO : Start of Moving Processed Files...
2021-05-01 17:57:57,849 : INFO : End of Moving Processed Files...
2021-05-01 17:57:57,849 : INFO : End of Data Load, validation and transformation
2021-05-01 17:57:57,850 : INFO : Start of Preprocessing...
2021-05-01 17:57:57,850 : INFO : Start of reading dataset...
2021-05-01 17:57:57,856 : INFO : End of reading dataset...
2021-05-01 17:57:57,856 : INFO : Start of feature encoding...
2021-05-01 17:57:57,887 : INFO : End of feature encoding...
2021-05-01 17:57:57,887 : INFO : Start of Droping Columns...
2021-05-01 17:57:57,888 : INFO : End of Droping Columns...
2021-05-01 17:57:57,888 : INFO : Start of finding missing values...
2021-05-01 17:57:57,889 : INFO : End of finding missing values...
2021-05-01 17:57:57,889 : INFO : Start of building final predictset...
2021-05-01 17:57:57,898 : INFO : End of building final predictset...
2021-05-01 17:57:57,899 : INFO : End of Preprocessing...
2021-05-01 17:57:57,899 : INFO : Start of Load Model
2021-05-01 17:57:57,899 : INFO : Model File KMeans loaded
2021-05-01 17:57:57,899 : INFO : End of Load Model
2021-05-01 17:57:57,902 : ERROR : Unsuccessful End of Prediction
Traceback (most recent call last):
  File "D:\GitHub\Employee-Retention-Prediction\App\apps\prediction\predict_model.py", line 60, in batch_predict_from_model
    kmeans = self.fileOperation.load_model('KMeans')
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 1201, in predict
    X = self._check_test_data(X)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 993, in _check_test_data
    X = check_array(X, accept_sparse='csr', dtype=[np.float64, np.float32],
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 72, in inner_f
    return f(**kwargs)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 650, in check_array
    raise ValueError("Found array with %d sample(s) (shape=%s) while a"
ValueError: Found array with 0 sample(s) (shape=(0, 11)) while a minimum of 1 is required.
